perceval/backends/core/confluence.py

```diff
@@ -94,13 +94,14 @@ def fetch(self, category=CATEGORY_HISTORICAL_CONTENT, from_date=DEFAULT_DATETIME
 
         return items
 
-    @staticmethod
-    def ancestors(item, content):
-        ancestors = []
-        if ('ancestors' in content):
+    def __fetch_ancestors(self, content):
```

Comment:
Since Perceval tries to manipulate as less as possible the data obtained from a data source, I would add all the data returned in `ancestors`